FlowCrypt/Functionality/Services/GlobalRouter.swift

```diff
@@ -134,7 +134,7 @@ extension GlobalRouter: GlobalRouterType {
             proceed(with: appContext.withSession(session))
         } catch {
             logger.logError("Failed to switch active user due to \(error.localizedDescription)")
-            handle(error: error, appContext: appContext)
+            hanleFatalError(error)
```

Comment:
Same here. What if we can not switch user because of DB error?

Reply:
modal + stay on current VC

Reply:
Let's throw it